### ISSUE TITLE
HTTP errors 405 "METHOD NOT ALLOWED" because of "Werkzeug"

### ISSUE THREAD

**Issue by Bjoernsen** [genotoul-bioinfo/dgenies#43]
**Describe the bug**
I am running D-Genies inside a docker container. Everything worked perfectly. But since a few days, I got many `method not allowed` http error messages. Most of the time, when I tried to upload files, but also sometimes for images, like `success.svg`.

I compared an old and new docker images and recognized that `werkzeug` was updated to `2.1.0` 2 days ago (2022-03-28). After switching back to `2.0.3`, everything works again.

**Screenshots**
![image](https://user-images.githubusercontent.com/4971650/160773227-70db9ed1-f4c3-4d69-ae29-29f6261e30ba.png)

**Computer (please complete the following information):**
 - OS: linux, docker
 - Browser: Chrome, Firefox, Edge
 - Version: latest

**Comment by pbordron** [comment]
Indeed, Since jinja2 <3 support was dropped from many current libraries, the D-Genies requirements need to be updated.

I will update the requirements.txt of D-Genies to fix it
